pontoon/base/views.py

```diff
@@ -1064,9 +1064,8 @@ def amagama(request):
         return HttpResponse("error")
 
 
-def transvision(request, repo, title):
+def transvision(request):
     """Get Mozilla translations from Transvision service."""
-    log.debug("Get Mozilla translations from Transvision service.")
```

Comment:
r+

On Sat, Nov 14, 2015 at 2:07 PM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/base/views.py
> <https://github.com/mozilla/pontoon/pull/269#discussion_r44858316>:
>
> >      """Get Mozilla translations from Transvision service."""
> > -    log.debug("Get Mozilla translations from Transvision service.")
>
> Some of the machinery services require HTTP instead of HTTPS and also API
> keys (none of which is true for Transvision). And to make things
> consistent, I've moved it to view as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/269/files#r44858316>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.